src/main/java/su/terrafirmagreg/core/mixins/common/gtceu/ToolHelperMixin.java

```diff
@@ -37,76 +25,4 @@ public abstract class ToolHelperMixin {
             charcoalPileBlock.onDestroyedByPlayer(state, world, pPos, player, true, state.getFluidState());
         }
     }
-
-    /**
-     * @author @jojo7682345
-     * @reason Changed the order of blocks broken from random to top to bottom
-     * Very specific things needed to be changed for this functions that are not able to be changes using mixins,
-     * therefore overwriting the entire thing is easier
-     */
-    @Overwrite
-    public static Set<BlockPos> iterateAoE(ItemStack stack, AoESymmetrical aoeDefinition, Level world,
```

Comment:
why was this removed? this made our aoe tools break blocks in a specific way to cause fewer collapses

Reply:
Gregtech AoE break logic was changed to no longer be random

Reply:
it wasn't random before, it was bottom-up for some reason -- but now that you mention it, I do remember one of the devs in the gt server asking me which way we had changed them to since they had to refactor that code for some other reason